requirements/base.in

```diff
@@ -9,4 +9,6 @@ ratelimit
 requests
 pypdf
 pycryptodome
+# Note(austin) higher versions need rust installed? pinning this until we address in the dockerfile
+safetensors<0.3.3
```

Comment:
It might make sense to put this in `constraints.in` instead since it's derived from other dependencies rather than something we depend on directly.

Reply:
i think you meant the unstructured repo?

Reply:
That makes sense - the api doesn't have a constraints file yet but good excuse to start one.

Reply:
Oh you're right, didn't pay attention ot which repo this was.